pkg/rc/replication_controller.go

```diff
@@ -1124,6 +1151,45 @@ func (rc *replicationController) finishTransfer(rcFields fields.RC, logger loggi
 	return nil
 }
 
+func (rc *replicationController) cancelTransfer(rcFields fields.RC, currentNodes []types.NodeName) error {
+	rc.nodeTransferMu.Lock()
+	close(rc.nodeTransfer.quit)
+	rc.nodeTransfer = nodeTransfer{}
+	rc.nodeTransferMu.Unlock()
+
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Minute)
+	defer cancel()
+
+	txn, cancelFunc := rc.newAuditingTransaction(ctx, rcFields, currentNodes)
+	defer cancelFunc()
+
+	err := rc.rcStatusStore.DeleteTxn(txn.Context(), rc.rcID)
+	if err != nil {
+		return util.Errorf("could not add RC status deletion to cancel node transfer transaction: %s", err)
+	}
+
+	ok, resp, err := txn.CommitWithRetries(rc.txner)
+	switch {
+	case err != nil:
+		err := util.Errorf(
+			"RC status deletion returned err after timeout: %s",
+			err,
```

Comment:
I think this `err` will try to reference the inner scope `err` but you mean to reference the outer scope `err` on line 1171.